starwars/src/components/PaginationButtons/PaginationButtons.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Button, Icon } from 'semantic-ui-react'
+
+const PaginationButtons = ({noOfContentsDisplayed,page, next,disableNext, prev,disablePrev}) =>{
+    // if(noOfContentsDisplayed === 0) return ""
```

Comment:
Don't forget to cleanup commented parts. Also, `noOfContentsDisplayed` is used nowhere. Get rid of extra parts in your app that don't do anything. Keep it simple 👍